gulp-tasks/utils/build-node-package.js

```diff
@@ -35,12 +35,5 @@ module.exports = (packagePath) => {
         },
       }],
     ],
-    plugins: [
-      // This ensures that the helpers used by Babel to accompany transpilation
-      // are only included in our Rollup bundles once, even if they're used
-      // in multiple source files.
-      // See https://github.com/rollup/rollup-plugin-babel#helpers
-      '@babel/transform-runtime',
```

Comment:
Not needed anymore?

Reply:
It shouldn't be. We target Node 10 now, and nothing in our code base ends up being transpiled into something that requires the Babel runtime.

(As far as I can tell!)